huaweicloud/services/dcs/resource_huaweicloud_dcs_instance.go

```diff
@@ -728,28 +751,28 @@ func resourceDcsInstancesRead(ctx context.Context, d *schema.ResourceData, meta
 				"backup_at":   backupPolicy.Policy.PeriodicalBackupPlan.BackupAt,
 			},
 		}
-		d.Set("backup_policy", bakPolicy)
+		mErr = multierror.Append(mErr, d.Set("backup_policy", bakPolicy))
 	}
 
 	// set tags
 	if resourceTags, err := tags.Get(client, "instances", d.Id()).Extract(); err == nil {
 		tagMap := utils.TagsToMap(resourceTags.Tags)
 		if err := d.Set("tags", tagMap); err != nil {
-			return fmtp.DiagErrorf("[DEBUG] Error saving tag to state for DCS instance (%s): %s", d.Id(), err)
+			return diag.Errorf("[DEBUG] error saving tag to state for DCS instance (%s): %s", d.Id(), err)
 		}
 	} else {
-		logp.Printf("[WARN] fetching tags of DCS instance failed: %s", err)
+		log.Printf("[WARN] fetching tags of DCS instance failed: %s", err)
 	}
 
 	// set white list
 	// some regions (cn-south-1) will fail to call the API due to the cloud reason
 	// ignore the error temporarily.
 	wList, err := whitelists.Get(client, d.Id()).Extract()
 	if err != nil || wList == nil {
-		logp.Printf("[WARN] Error fetching whitelists for DCS instance, error: %s", err)
+		log.Printf("[WARN] Error fetching whitelists for DCS instance, error: %s", err)
 	}
 
-	logp.Printf("[DEBUG] Find DCS instance white list : %#v", wList.Groups)
+	log.Printf("[DEBUG] Find DCS instance white list : %#v", wList.Groups)
```

Comment:
according to line 771, if the `wList` is nil, here will cause panic.

Reply:
fixed